app/utils/ViewUtils.scala

```diff
@@ -27,12 +27,14 @@ object ViewUtils {
       ("HMRC-MTD-VAT", mgs("tax-service.vat")),
       ("HMRC-CGT-PD", mgs("tax-service.cgt")),
       ("HMRC-PPT-ORG", mgs("tax-service.ppt")),
-      ("TRUST", mgs("tax-service.trusts")) // TODO update to HMRC-TERS
+      ("TRUST", mgs("tax-service.trusts")), // TODO update to HMRC-TERS
     )
-
-    if(appConfig.cbcEnabled) {
-      filters ++ Seq(("HMRC-CBC", mgs("tax-service.cbc")))
-    } else filters
+    (appConfig.cbcEnabled,appConfig.pillar2Enabled)  match {
+      case (true, true) => filters ++ Seq(("HMRC-CBC", mgs("tax-service.cbc")), ("HMRC-PILLAR2-ORG", mgs("tax-service.pillar2")))
+      case (true, false) => filters ++ Seq(("HMRC-CBC", mgs("tax-service.cbc")))
+      case (false, true) => filters ++ Seq(("HMRC-PILLAR2-ORG", mgs("tax-service.pillar2")))
+      case _ => filters
+    }
```

Comment:
I would do that way or something like that. Easier to add new tax 
      ("TRUST", mgs("tax-service.trusts")) // TODO update to HMRC-TERS
    ) ++ 
      (if(appConfig.cbcEnabled) Seq(("HMRC-CBC", mgs("tax-service.cbc"))) else Seq.empty) ++ 
      (if(appConfig.pillar2Enabled) Seq(("HMRC-PILLAR2-ORG", mgs("tax-service.pillar2"))) else Seq.empty)